### PR TITLE
Add block break effect to area pickaxe

### PR DESCRIPTION
**Pull request by Owen1212055**
## Overview
Adds block break effect to area pickaxe.

## Description
Adds block break effect to area superpickaxe to match upstream/recursive pickaxe behavior.

I'm not sure what to do here as the commits that were responsible for adding this were reverted on a merge a _while_ back.
Added in: https://github.com/IntellectualSites/FastAsyncWorldEdit/commit/96e56bdd0cbf5d7640fa30984f78c27a084b6d05
Reverted in: https://github.com/IntellectualSites/FastAsyncWorldEdit/commit/66744bfaa4154569176e677ebb71a36470147d8b

## Submitter Checklist

- [X ] Make sure you are opening from a topic branch (**/feature/fix/docs/ branch** (right side)) and not your main branch.
- [X] Ensure that the pull request title represents the desired changelog entry.
- [] New public fields and methods are annotated with `@since TODO`.
- [X] I read and followed the [contribution guidelines](https://github.com/IntellectualSites/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by NotMyFault** [comment]
> I'm not sure what to do here as the commits that were responsible for adding this were reverted on a merge a while back.

If there's no downside of adding it again and no associated issue, I guess we're good to go to reintegrate that?
Either it has been removed on purpose or on accident when local changes were different to upstream changes.

**Comment by Owen1212055** [comment]
Yeah, it looked like it may have been removed by accident.

**Review by NotMyFault — APPROVED** [review]

File: worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/AreaPickaxe.java [hunk 1]
```diff
@@ -76,6 +78,9 @@ public boolean actPrimary(Platform server, LocalConfiguration config, Player pla
 
                             editSession.setBlock(x, y, z, BlockTypes.AIR.getDefaultState());
 
+                            BlockVector3 pos = BlockVector3.at(x, y, z);
+                            ((World) clicked.getExtent()).queueBlockBreakEffect(server, pos, initialType,
+                                    clicked.toVector().toBlockPoint().distanceSq(pos));
```
Comment (dordsor21) [inline worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/AreaPickaxe.java:83]: This should probably only be done if the setBlock method returns true

Comment (Owen1212055) [reply worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/AreaPickaxe.java:83]: It should be noted upstream doesn't check for this, but sure.

Comment (dordsor21) [reply worldedit-core/src/main/java/com/sk89q/worldedit/command/tool/AreaPickaxe.java:83]: In that case also ensure to wrap in //FAWE start and end comments

**Pull request merged**